api/src/config/publication.js

```diff
@@ -12,13 +12,15 @@ const playwrightConfig = {
   sortBySuffix: '&view_op=list_works&sortby=pubdate', // put most recent pubs first
```

Comment:
Also, the filename `playwright.js` makes more sense to me IMO. I'd like to get a second opinion though. @vgoh1

Reply:
Agree, I prefer this file being called `playwright.js`. Maybe we can move categoryType and categoryTypeEnum to a config file named `publication.js` though?